README.md

```diff
@@ -138,3 +138,24 @@ For more locally testable scripts, Cram-style functional tests live in `tests` a
 
 1. Download Cram: `pip install cram`
 2. Run the tests: `cram tests/`
+
+## Working on this repo
+
+This repo is configured to use [pre-commit](https://pre-commit.com),
+to help automatically catch common coding errors and syntax issues
+with changes before they are committed to the repo.
+.
+If you will be writing new code or otherwise working within this repo,
+please do the following to get started:
+
+1. install `pre-commit` by running either `python -m pip install
+   pre-commit` or `brew install pre-commit`, depending on your
+   preferred package management solution
```

Comment:
Suggestion: link to the official docs for all the options (some were missing), plus the benefit of future-proofing


```suggestion
1. [install `pre-commit`](https://pre-commit.com/#install)
```

Reply:
addressed, thanks.